css/collection-card.styl

```diff
@@ -1,26 +1,53 @@
 .collections-card-list
   @extends $owned-card-list
-  
+
 .collection-card
   @extends $owned-card
   background: url('/assets/simple-pattern.png') center center repeat
   border-radius: 1em
```

Comment:
Should the borders still be rounded? 

<img width="253" alt="screen shot 2017-11-10 at 4 42 08 pm" src="https://user-images.githubusercontent.com/5016731/32681648-1ed85e2e-c636-11e7-8fe4-41e93b632c74.png">